### PR TITLE
Consistency and clean up

### PR DESCRIPTION
**Pull request by hhsecond**
Implementing #31 and circleCI modifications
- Decoding all the results from RedisAI now
- Return type is dict now (unless the reply is OK)
- Enums are removed. User can (must) pass strings for dtype, device and backend arguments
- Utils.py is much cleaner now
- Bonus: Implemented debug utility to print out the commands if enabled

### CODE REVIEW

**Comment by hhsecond** [comment]
Test cases are passing in my local; it's failing probably because the edge image is built at night?

File: test/test.py [hunk 1]
```diff
@@ -194,18 +239,24 @@ def test_model_list(self):
...
+        con.modelset('m', 'tf', 'cpu', model_pb,
+                     inputs=['a', 'b'], outputs=['mul'], tag='v1.2')
         model_path = os.path.join(MODEL_DIR, 'pt-minimal.pt')
         ptmodel = load_model(model_path)
         con = self.get_client()
-        con.modelset("pt_model", Backend.torch, Device.cpu, ptmodel)
-        mlist = con.modelist()
-        self.assertEqual(mlist, [[b'pt_model', b''], [b'm', b'v1.2']])
+        con.modelset("pt_model", 'torch', 'cpu', ptmodel)
+        mlist = con.modescan()
```
Comment (lantiga) [inline test/test.py:248]: modescan -> modelscan

Comment (lantiga) [reply test/test.py:248]: I would still add the underscore maybe?

Comment (hhsecond) [reply test/test.py]: IMO that's not a pythonic way of handling experimental APIs. I have added a warning on the function call. Wouldn't that be enough?
https://github.com/RedisAI/redisai-py/pull/32/files#diff-0f59821ab2985447357ebcef5825b8e5R110

Comment (hhsecond) [reply test/test.py]: stupid me (talking about the spelling correction)

**Pull request merged**